doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+
+Some frameworks such as TensorFlow have fused (or layer) ops. nGraph provides
+optional operations (listed in the following table) to help users easily
+translate (map) any quantized model created from frameworks with fused ops to
+nGraph.
+
+.. table:: Experimental Quantized Ops (optional)
+
+
+	+-----------------------------------+----------------------------------+
+	| Operator                          | Description                      |
+	+===================================+==================================+
+	| QuantizedConvolutionBias          | Performs quantized convolution   |
+	|                                   | with bias                        |
+	+-----------------------------------+----------------------------------+
+	| QuantizedConvolutionBiasAdd       | Performs quantized convolution   |
+	|                                   | with bias and add                |
+	+-----------------------------------+----------------------------------+
+	| QuantizedConvolutionBiasSignedAdd | Performs quantized convolution   |
+	|                                   | with bias and signed add         |
+	+-----------------------------------+----------------------------------+
+	| QuantizedConvolutionRelu          | Performs quantized convolution   |
+	|                                   | and ReLu                         |
+	+-----------------------------------+----------------------------------+
+	| QuantizedDotBias                  | Performs quantized dot with bias |
+	+-----------------------------------+----------------------------------+
+
+nGraph Quantization Design
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+The goal of nGraph quantization is to flexibly support a wide variety of
+frameworks and users. The use of scale and zero point as well as quantized
+builders in the nGraph design helps to achieve this goal.
+
+Scale and Zero Point
+********************
+Using scale and zero point allows nGraph to be framework agnostic (that is, it
+can equally support all deep learning frameworks). nGraph bridges will
+automatically convert min and max (provided by a DL framework) to scale and zero
+point as needed. Quantized builders are available to help the bridges perform
+this calculation. However, if users are directly using nGraph (i.e., not using a
+bridge), they are required to provide scale and zero point for quantized ops.
+
+Another advantage of using scale and zero point to express quantization
+parameters is that users can flexibly implement quantized ops into various
+nGraph backends. When implementing quantized ops, nGraph backends can directly
+use scale and zero point instead of min and max.
+
+Quantized Builders
+******************
+Quantized builders are helper utilities to assist framework integrators to
+enable quantized models with nGraph. They serve as an API (interface) between
+framework bridges and nGraph, allowing framework bridges to directly construct
+ops in nGraph core IR.
```

Comment:
allowing framework bridges to directly construct ops in the nGraph Abstraction Layer